src/plugins/data/config.ts

```diff
@@ -48,6 +48,9 @@ export const configSchema = schema.object({
         enabled: schema.boolean({ defaultValue: false }),
       }),
     }),
+    usageTelemetry: schema.object({
+      enabled: schema.boolean({ defaultValue: false }),
```

Comment:
Should this be true since currently it's always on? I can see the argument though about defaulting to false. However, since we are going to `2.0.0` now's the time to change the default behavior.

Reply:
Thanks Rocky, this change will target 2.0.0 and it is not feature break, we still give customer ability to turn it on if they need. we will keep the default to false and document it when release it to managed service or 2.2.0 release.

Reply:
Yes. the default behavior is changed, but it is target 2.0.0, we will document it and customer still have ability to turn it on if they needed. Thanks.

Reply:
hi @Flyingliuhub, is this the file that the customer can disable "data plugin" as per 
"data plugin config enabled value" mentioned in the doc issue [494](https://github.com/opensearch-project/documentation-website/issues/494)?
I just want to confirm that i tell them the exact location to change to true or false.

Reply:
Customer cannot change the value from configuration.ts, but they can use opensearch_dashboards.yml 's value to override the value here.

Reply:
oh ok thanks. I got confused by the table with entry for "data plugin" true or false. seemed to indicate they would need to change it. 
got it! will only show to enable, change opensearch_dashboards.yml `data.search.usageTelemetry.enabled` setting to `true`